nvflare/edge/assessors/model_update.py

```diff
@@ -54,44 +37,68 @@ def __init__(
         max_model_history,
         device_selection_size,
         min_hole_to_fill=1,
+        global_lr=1.0,
+        staleness_weight=False,
         device_reuse=True,
         const_selection=False,
     ):
-        """
+        """Initialize the ModelUpdateAssessor.
         Enable both asynchronous and synchronous model updates from clients.
-        For asynchronous updates, the staleness is calculated based on the current model version.
-        And the aggregation weights are calculated following FedBuff paper "Federated Learning with Buffered Asynchronous Aggregation".
+        For asynchronous updates, the staleness is calculated based on the starting and current model version.
+        And the aggregation scheme and weights are calculated following FedBuff paper "Federated Learning with Buffered Asynchronous Aggregation".
+
+        Args:
+            persistor_id (str): ID of the persistor component used to load and save models.
+            num_updates_for_model (int): Number of updates required before generating a new model version.
+            max_model_version (int): Maximum number of model versions allowed in the workflow.
+            max_model_history (int): Maximum number of historical model versions to keep in memory.
```

Comment:
These 3 args (num_updates_for_model, max_model_version, max_model_history) are related to model management, hence should belong to ModeManager, not here.

Reply:
updated